ai-platform-unified/notebooks/official/pipelines/control_flow_kfp.ipynb

```diff
@@ -1,820 +1,1037 @@
 {
```

Comment:
<p>A few comments on this setup (that probably hold for all the notebooks):</p><ul><li>this won't do the right thing on colab (at this point they haven't auth'd yet, and after they do, the gcloud call will return their email address)</li><li>If they haven't run the previous cell (say b/c they don't know the service account), then this cell will give a <code>NameError: name 'SERVICE_ACCOUNT' is not defined</code> error.</li><li>There's a section below where they create a service account if they need to-- I wonder if that should come before this section.</li><li>Because of the above, and because a given project may have many service accounts, I think the text in this section needs to give a bit more context as to what service account they should fill in/use.</li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/411/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>